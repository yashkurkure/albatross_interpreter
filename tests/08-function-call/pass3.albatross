var f1 int := 0;
var f2 int := 0;

fun f1 int () {
  return 1;
}

fun f2 int () {
  return 1;
}

return f1()+f1+f2()+f2;
