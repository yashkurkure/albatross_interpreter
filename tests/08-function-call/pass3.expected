Variable declared "f1" type int
Variable declared "f2" type int
Function declared "f1" returns int
Function declared "f2" returns int
Function called "f1" returns int
Variable read "f1" type int
Function called "f2" returns int
Variable read "f2" type int
